common.js

```diff
@@ -0,0 +1,142 @@
+/* eslint no-console:0 */
+//
+// Use a Custom Resource Definition to extend the Kubernetes API and the client.
+//
+var exports = module.exports = {};
+let fs = require('fs');
+const path = require('path');
+
+exports.delete_env = async function (client, env_name, nmspace) {
+
+    const del_env = await client.apis['fission.io'].v1.namespaces(nmspace).environments(env_name).delete();
+    console.log("Fission envirnment ",del_env['body']['details']['name']," deleted");
```

Comment:
before using multi level keys we should check whether it exist or not otherwise it will break. Like you have used del_env['body']['details']['name'], what if any of the key doesn't exist, it will give error.

Reply:
every environment in fission has a name and secondly if it doesnt exist then fission will give an error message which will be stored in that variable

Reply:
ok